src/components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useMocks } from "../../utils/hooks/useMocks";
+import "./SideBar.styles.css";
+
+function SideBar({ children, selectedOptions, setSelectedOptions }) {
+  const { categories } = useMocks();
+
+  const isSelected = (id) => {
+    if (selectedOptions.find((option) => option === id)) {
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
the function is checking in the value is included in an array, there is a method called `includes` that returns true if the value is in the array, false otherwise.

```suggestion
  const isSelected = id => selectedOptions.includes(id)
```